libs/logstash-bridge/src/main/java/org/elasticsearch/logstashbridge/common/Settings.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+package org.elasticsearch.logstashbridge.common;
+
+import org.elasticsearch.logstashbridge.StableAPI;
+
+public class Settings extends StableAPI.Proxy<org.elasticsearch.common.settings.Settings> {
```

Comment:
Naming these the exact same as the classes in server presents problems for quickly opening the files in intellij. Can we please have all of the proxies with qualified names?